docusaurus/docs/operate/upgrades/2_upgrade_procedure.md

```diff
@@ -37,7 +37,7 @@ This process involves several key steps:
 2. **Implementation**: The proposed changes are implemented in the codebase.
 3. **Testing**: Thorough testing of the proposed changes is conducted in devnet and testnet environments before mainnet deployment.
 4. **Announcement**: Upon successful testing, we announce the upgrade through our social media channels and community forums.
-5. **Deployment**: An upgrade transaction is sent to the network, allowing node operators using [Cosmovisor](../../operate/walkthroughs/full_node_walkthrough.md) to automatically upgrade their nodes at the specified block height.
+5. **Deployment**: An upgrade transaction is sent to the network, allowing node operators using [Cosmovisor](../walkthroughs/full_node_walkthrough.md) to automatically upgrade their nodes at the specified block height.
```

Comment:
Were these links broken previously?

Wdyt about an issue (I don't mind opening it) to automate broken link checking?

Reply:
> Wdyt about an issue (I don't mind opening it) to automate broken link checking?

It is already be done by our docuseries CI.

Not sure how it got through.